src/class-elements/syntax/valid/cls-decl-elements-valid-syntax-export-default.template

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-ClassElement
+name: class declaration module default export
+path: language/statements/class/elements/syntax/valid/export-default-
+flags: [module]
+features: [class]
+---*/
+
+export default class C /*{ heritage }*/{
+  /*{ elements }*/
+}
```

Comment:
Maybe this is the only compelling template in this PR, IMO as the class declaration has an odd variation that is not found elsewhere.

In this case, the variation would be defining a declaration without a BindingIdentifier. This would be a production I believe that is good enough for checking syntax.

In this case, I would just change this template code to be like:

```
export default class /*{ heritage }*/{
  /*{ elements }*/
}
```